src/base/http/types.h

```diff
@@ -64,16 +64,19 @@ namespace Http
         QByteArray data; // File data
     };
 
-    struct Request
```

Comment:
Just create Request struct with `RawStringMap gets`. As there can be any binary data according to the standard, it is fully consistent with the model (no need to complicate it). As I said, parsing of these data is the task of the application. In our case, we can consider each Action (eg. `action_query_torrents()`) as a separate application, so we should parse GET params in these actions.